1-exercises/B-while-loop/exercise.js

```diff
@@ -4,10 +4,17 @@
     Using a while loop, complete the function below so it logs (using console.log) the first n even numbers as a comma-seperated string.
     The list of numbers should start with 0. n is being passed in as a parameter.
 */
-
 function evenNumbers(n) {
-    // TODO
-}
+    let arrayOfEvenNumbers = [];
```

Comment:
Hi, Emeka I hope you are ok, I reviewed your answer and it is correct and functional.